lsinitrd.sh

```diff
@@ -347,7 +347,7 @@ case $bin in
                 echo "Early CPIO image"
                 list_files
             fi
-            if [[ -d "$dracutbasedir/src/skipcpio" ]]; then
+            if [[ -f "$dracutbasedir/src/skipcpio/skipcpio" ]]; then
```

Comment:
Perhaps `-x` is better here than `-f` ?

Reply:
well it depends on whether we only care about the presence of the file or if we also need to check for executable? I'm not sure it matters much in this case, so `-f` is fine?

Reply:
Few lines below `-x` is checked for the same files

`if ! [[ -x $SKIP ]]; then1

Reply:
Ah, then yes, let's simplify here and use `-x`. It gives a _small_ speedup to simplify.

Reply:
I've opted for `-f` as there's already `-x` later in the code. I'd rather avoid refactoring :) - at least from my side.

Reply:
My suggestion was simply use `-x` without additional refactoring (check for `-x` twice) to help to reason about the code, but the difference is marginal.